src/DeploymentPipelines/IndividualPipelineStack/index.ts

```diff
@@ -241,4 +297,23 @@ export class IndividualPipelineStack<
     commands.push(synthCommand);
     return commands;
   }
+  /**
+   * Linux commands for assuming a role.
+   *
+   * @private
+   * @param {string} [role]
+   * @returns {string[]}
+   */
+  private createAssumeRoleCommands(role: string) {
+    return role
+      ? [
+          `ASSUME_ROLE_ARN="arn:aws:iam::$account_id:role/${role}`,
```

Comment:
replace `aws` in this ARN with the variable for `partition`. Other regions like govcloud are in a different partition eg. `aws-gov`. Also is `$account_id` a typo or intentional?